### ISSUE TITLE
Docs: Icons Canvas icon names are sometimes cropped and cause accessibility test to fail.

### ISSUE THREAD

**Issue by coliff** [trimble-oss/modus-web-components#2103]
### Prerequisites

- [X] I have [searched](https://github.com/trimble-oss/modus-web-components/issues?q=is%3Aissue) for duplicate or closed issues
- [X] I have read the [contributing guidelines](https://github.com/trimble-oss/modus-web-components/blob/main/CONTRIBUTING.md)

### Describe the issue

Icon names are sometimes cropped and cause accessibility test to fail. We should change that box wrapper from `width: 100px` to `min-width: 100px`

### Reduced test cases

Check URL: https://modus-web-components.trimble.com/?path=/story/components-icons--default&args=name:battery_charging_horizontal

![image](https://github.com/trimble-oss/modus-web-components/assets/1212885/861fb12c-1afe-47d0-a25b-f03a3171fd9a)

### What operating system(s) are you seeing the problem on?

Windows

### What browser(s) are you seeing the problem on?

Microsoft Edge

### What is the issue regarding ?

Modus Web Components Website (Storybook)

### What version of npm package are you using ?

v.0.13.0

### Priority

Low

### What product/project are you using Modus Components for ?

Modus

### What is your team/division name ?

Modus

### Are you willing to contribute ?

Yes

### Are you using Modus Web Components in production ?

https://modus-web-components.trimble.com/
